backend/experiment/management/commands/createexperiment.py

```diff
@@ -0,0 +1,122 @@
+import os.path
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Creates a new experiment rules class'
+
+    def handle(self, *args, **options):
+        # Ask for the experiment name
+        experiment_name = input("What is the name of your experiment? (ex. Musical Preferences): ")
+
+        # Create the experiment rule class
+        self.create_experiment_rule_class(experiment_name)
+
+        # Add the new experiment to ./experiment/rules/__init__.py
+        self.register_experiment_rule(experiment_name, './experiment/rules/__init__.py')
+
+        # Create a basic test file for the experiment
+        self.create_test_file(experiment_name)
+
+    def create_experiment_rule_class(self, experiment_name):
+        # Get the experiment name in different cases
+        experiment_name_snake_case, experiment_name_snake_case_upper, experiment_name_pascal_case = self.get_experiment_name_cases(experiment_name)
+
+        # Create a new file for the experiment class
+        filename = f"./experiment/rules/{experiment_name_snake_case}.py"
+
+        # Check if the file already exists
+        if os.path.isfile(filename):
+            # Warn the user that the file already exists and ask if they want to overwrite it
+            self.stdout.write(self.style.WARNING(f"File {filename} already exists"))
+            overwrite = input("Do you want to overwrite it? (y/n): ")
+
+            # If the user does not want to overwrite the file, exit the command
+            if overwrite.lower() != 'y':
```

Comment:
I worry a bit that giving an option to overwrite may break existing experiments in the database. I think it's safer not to provide that option.